.github/workflows/build-pr-preview-test.yml

```diff
@@ -0,0 +1,323 @@
+name: Build Preview Test
+run-name: 'Build Preview for "${{ github.event.pull_request.title }}" (#${{ github.event.pull_request.number }})'
+
+on:
+  pull_request:
```

Comment:
Used for testing as `pull_request_target` actions/files will only update from changes on `main`.